src/core/linalg/src/sparse/4C_linalg_map.hpp

```diff
@@ -27,11 +28,29 @@ FOUR_C_NAMESPACE_OPEN
 
 // NOLINTBEGIN(readability-identifier-naming)
 
+
+
 namespace Core::LinAlg
 {
-
   class Map
   {
+    // This wrapper may have two different variants
+    // Either it holds a "view" to the Epetra_Map or Epetra_BlockMap
```

Comment:
```suggestion
    // Either it holds an  Epetra_Map or Epetra_BlockMap
```

Minor: I wouldn't call this "view" here.